docs/stronghold.md

```diff
@@ -0,0 +1,38 @@
+---
+id: stronghold
+title: Stronghold
+sidebar_label: Stronghold
+---
+
+Use Stronghold to have your earnings paid out in US dollars.
+
+> You won't see your payments immediately if the amount you've received is
```

Comment:
can we give the exact threshold here? or an approximation even

Reply:
Added approximation.